setup.py

```diff
@@ -15,7 +15,8 @@
     url='https://github.com/davidhamann/python-fmrest',
     packages=['fmrest'],
     include_package_data=True,
-    install_requires=['requests>=2'],
+    install_requires=['requests>=2',
+                      'pycognito>=0.1.4'],
```

Comment:
We should make `pycognito` and optional dependency, so that it is not installed, when Cloud is not used.

I would suggest something like:

```
    install_requires=['requests>=2'],
    extras_require={
            'cloud': ['pycognito>=0.1.4']},
```

Reply:
Thank you! This has been updated.